src/styles/_inline-tiles.scss

```diff
@@ -0,0 +1,3 @@
+#nosto-ugc-container {
+    background-color: var(--widget-background);
```

Comment:
Do we need this? I think it's a bit of an overkill introducing a common CSS variable and a SCSS partial with only one styling. 
This style needs to be included in all the widget type similar to how `background-color: var(--widget-background);` will be included (if this partial is not provided). Do you see any particular advantage of this?